logger/logger_config.go

```diff
@@ -44,11 +46,16 @@ func (l Level) String() string {
 }
 
 type Config struct {
-	Level Level `json:",omitempty"`
+	Level  Level  `json:",omitempty"`
+	Custom Logger `json:"-"`
+}
+
+func (c *Config) SetLogger(customLogger Logger) {
+	c.Custom = customLogger
```

Comment:
I think we don't need this method, since it doesn't make setting the logger easier.

Reply:
Agree, removed it as you suggested